src/lao.c

```diff
@@ -323,6 +353,14 @@ static int l_is_big_endian(lua_State *L)
 	return 1;
 }
 
+static int l_append_global_option(lua_State *L)
+{
+	const char *key = luaL_checkstring(L, 1);
+	const char *val = luaL_checkstring(L, 2);
```

Comment:
`default_driver=alsa`  has to be a string, debug, verbose and quiet could be bool, but I think we should insist on 'yes' or 'no', and document that.  The libao routine takes (const char *key, const char *value). How it interprets the empty string I think we should leave in its hands.